contracts/multisig/src/types.rs

```diff
@@ -33,42 +32,10 @@ pub enum MultisigState {
     },
 }
 
-const MESSAGE_HASH_LEN: usize = 32;
 
-impl TryFrom<HexBinary> for MsgToSign {
-    type Error = ContractError;
-
-    fn try_from(other: HexBinary) -> Result<Self, Self::Error> {
-        if other.len() != MESSAGE_HASH_LEN {
-            return Err(ContractError::InvalidMessageFormat {
-                reason: "Invalid input length".into(),
-            });
-        }
-
-        Ok(MsgToSign(other))
+impl From<HexBinary> for MsgToSign {
+    fn from(value: HexBinary) -> Self {
+        MsgToSign(value)
```

Comment:
this circumvents previous assumptions about MsgToSign, I want to discuss this in more detail before approving this change